kuma/static/js/auth.js

```diff
@@ -54,71 +54,75 @@
+        (function() {
+            try {
+                var $browserRegister = $('#browser_register');
+                var matchKey = 'account-match-for';
+                var matchCurrent = $browserRegister.data(matchKey);
+                var matchStored = localStorage.getItem(matchKey);
+
+                // The user is on the registration page and has been notified that
+                // there is an MDN profile with a matching email address.
+                if(matchCurrent && !matchStored) {
+                    localStorage.setItem(matchKey, matchCurrent);
+                }
+
+                // After seeing the notice, the user tried to sign in with a second
+                // social account, but was again directed to the registration page.
+                // This will happen if the second social account was also not
+                // tied to an MDN profile.
+                else if(!matchCurrent && matchStored && $browserRegister.length) {
+                    localStorage.removeItem(matchKey);
+                }
+
+                // After seeing the notice, the user took an action that resulted in
+                // them landing on another page. Either the user abandoned the
+                // registration process or the user tried to sign in with a second
+                // social account and was successful. In either case, the
+                // localStorage item should be immediately removed.
+                else if(!matchCurrent && matchStored) {
+                    $doc.on('mdn:login', function() {
+                        mdn.Notifier.growl('You can now use ' + matchStored + ' to sign in to this MDN profile.', { duration: 0, closable: true }).success();
+                    });
+                    localStorage.removeItem(matchKey);
+                }
 
-            // After seeing the notice, the user tried to sign in with a second
-            // social account, but was again directed to the registration page.
-            // This will happen if the second social account was also not
-            // tied to an MDN profile.
-            else if(!matchCurrent && matchStored && $browserRegister.length) {
-                localStorage.removeItem(matchKey);
             }
-
-            // After seeing the notice, the user took an action that resulted in
-            // them landing on another page. Either the user abandoned the
-            // registration process or the user tried to sign in with a second
-            // social account and was successful. In either case, the
-            // localStorage item should be immediately removed.
-            else if(!matchCurrent && matchStored) {
-                $doc.on('mdn:login', function(service) {
-                    mdn.Notifier.growl('You can now use ' + matchStored + ' to sign in to this MDN profile.', { duration: 0, closable: true }).success();
-                });
-                localStorage.removeItem(matchKey);
+            catch (e) {
```

Comment:
Did `eslint` no complain about the unused variable `e`? Safe to remove as we are not using it.

Reply:
I think if we don't use it it will bubble up. The linter did not object.